### ISSUE TITLE
[PAD0] Misconfigured device, check GUIDs.

### ISSUE THREAD

**Issue by Despairness** [x360ce/x360ce#197]
Hello!

Everytime I want to setup x360ce such error appears. I was googling a lot to find solution for this but so far so nothing...
Before above error I had another one: failed to load xinput1 3 and I found this threads:
http://steamcommunity.com/discussions/forum/11/648817378045289297/?l=polish
https://gbatemp.net/threads/issues-with-windows-8.363307/
and few others. 

So I installed what was recommended and then appears: Misconfigured...

Everything was fine on Win7 but after update do Win10 something gone wrong.

Any ideas?

Cheers!

**Comment by Squall-Leonhart** [comment]
GUID will be different on windows 10 then it was on 7.

**Comment by Despairness** [comment]
So there's nothing I can do with that? To be honest without x360ce I feel like without right hand.

About my instalation, I got links from here: https://github.com/x360ce/x360ce/issues/54 and that fixed my failed to load... issue.

Maybe I'm a little blind but there's no simillar issue as mine.
I just downloaded x360ce from here http://ngemu.com/threads/x360ce-3-0-1-6-beta-libraries-3-4-1-1173.170945/ but still wild error appears.

Cheers.

**Comment by Squall-Leonhart** [comment]
what it means is that you must run the configuration utility and reconfigure your gamepad, even if it means deleting the existing ini.

You cannot just move the configuration ini from one computer to another, or between windows updates which will mess with device GUID's.

**Comment by Despairness** [comment]
I know that sir but this error appears right after those two windows come out (create files). I can't even setup one button for my gamepad and that's why this problem annoys me so much.

Let me show you my story via screens.

I launch x360ce then:
![1st](https://cloud.githubusercontent.com/assets/13669161/9204349/004d20d6-405d-11e5-8884-5eb8878a9b15.png)
Yes.

![2nd](https://cloud.githubusercontent.com/assets/13669161/9204351/06620e96-405d-11e5-8e07-a3889d2a07fa.png)
Yes.

(now this button with ready settings)
Cancle.

![3rd](https://cloud.githubusercontent.com/assets/13669161/9204356/125ad7e6-405d-11e5-94d3-c802ea2ebe18.png)
And this appear everytime I try to do it or when I try to setup button.

**Comment by lanrius** [comment]
Hi

I have exactly the same issue. Using Windows 7. I've spent days trying to figure out a solution but no luck on this yet. 

Any help guys?

**Comment by Squall-Leonhart** [comment]
ok, im gunna have to test this... which means updating xbcd to work on windows 10 in my VM build.

**Comment by Squall-Leonhart** [comment]
This issue is caused by an incompatibility between device naming/instances on windows 10 and prior versions of windows.

I noticed that on windows 10 my "MLG Gamepad for Xbox 360 (Controller)" becomes (Controller) MLG Gamepad for Xbox 360.

The solution is to copy all the information from the old gamepad's guid to the new guid in the ini.

**Comment by EronzG** [comment]
Hello all. I just resolved the "[PAD0] Misconfigured device, check GUIDs" error on my PC. I simply deleted the x360ce_x64 application, then also deleted the x360ce configurations settings file. Then downloaded the x360ce_x64 application from https://www.x360ce.com/ and placed it in the game files folder where I had earlier deleted the old one. Upon launching the x360 app, a new configuration file was automatically created. 
![image](https://user-images.githubusercontent.com/25104615/83574396-52d0a580-a525-11ea-8559-e86426400566.png)

When I initially deleted only the x360ce app, I kept getting the error. It was only after I deleted the configurations file that it worked
